CodeGen/Generators/UnitsNetGen/UnitTestBaseClassGenerator.cs

```diff
@@ -45,37 +45,38 @@ internal class UnitTestBaseClassGenerator : GeneratorBase
         /// </summary>
         private readonly string _numberSuffix;
 
-        public UnitTestBaseClassGenerator(Quantity quantity)
+        public UnitTestBaseClassGenerator( Quantity quantity )
```

Comment:
Nitpick, but no whitespace inside ()'s is more consistent with the code base. It seems to be introduced several places in this file, probably auto formatted.

Reply:
Ah yes, stupid VS auto-formatting. Removed.